KarhooUISDKTests/TestCases/Screens/CheckoutScreen/CheckoutViewModelSpec.swift

```diff
@@ -305,6 +312,42 @@ class KarhooCheckoutViewModelSpec: QuickSpec {
                         expect(sut.loyaltyViewModel.tripAmount).to(equal(mockLoyaltyUIModel.tripAmount))
                         expect(sut.loyaltyViewModel.burnAmount).to(equal(mockLoyaltyUIModel.burnAmount))
                     }
+                    
+                    context("and when booking state is changed to .success") {
+                        beforeEach {
+                            mockBookingWoker.stateSubject.send(.success(mockTripInfo))
+                        }
+                        
+                        it ("should pass correct data to router success scene") {
+                            expect(mockCheckoutRouter.routeSuccessSceneCalledData?.loyaltyInfo.shouldShowLoyalty).to(equal(mockLoyaltyWorker.isLoyaltyEnabled))
+                            expect(mockCheckoutRouter.routeSuccessSceneCalledData?.loyaltyInfo.loyaltyPoints).to(equal(mockBasicLoyaltyInfo.loyaltyPoints))
+                            expect(mockCheckoutRouter.routeSuccessSceneCalledData?.loyaltyInfo.loyaltyMode).to(equal(LoyaltyMode.earn))
+                            
+                        }
+                    }
+                    context("and when is switched to burn mode") {
+                        beforeEach {
+                            sut.loyaltyViewModel.isBurnModeOn = true
+                            mockBasicLoyaltyInfo = .init(
+                                shouldShowLoyalty: true,
+                                loyaltyPoints: 10,
+                                loyaltyMode: .burn
+                            )
+                            mockLoyaltyWorker.getBasicLoyaltyInfoToReturn = mockBasicLoyaltyInfo
+                        }
+                        context("and when booking state is changed to .success") {
+                            beforeEach {
+                                mockBookingWoker.stateSubject.send(.success(mockTripInfo))
+                            }
+                            
+                            it ("should pass correct data to router success scene") {
+                                expect(mockCheckoutRouter.routeSuccessSceneCalledData?.loyaltyInfo.shouldShowLoyalty).to(equal(mockLoyaltyWorker.isLoyaltyEnabled))
```

Comment:
looks like we have some formatting issues here

Reply:
Only according to GitHub. In IDE looks ok.